common/src/test/java/org/esbtools/message/admin/common/ErrorServiceTest.java

```diff
@@ -23,6 +23,8 @@
 import java.util.ArrayList;
```

Comment:
A few suggestions for refactoring the tests in this class.  
  * Split out multiple actions into multiple tests, ideally with only 1 arrange/act/assert sequence per test (i.e. testPartialViewMessages() becomes testPartialViewXMLMessage() includes lines 196 - 213 and testPartialViewJSONMessage() includes lines 215 - 228)
  * Name tests to indicate the scenario they are testing and expected outcome (i.e. testPartialViewXMLMessages() becomes testSensitiveInfoRemovedFromXMLMessage() and testPartialViewJSONMessage() becomes testSensitiveInfoRemovedFromJSONMessage())